tensorflow/lite/micro/kernels/neg.cc

```diff
@@ -31,15 +31,22 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
   const TfLiteTensor* input = GetInput(context, node, kInputTensor);
   TfLiteTensor* output = GetOutput(context, node, kOutputTensor);
   switch (input->type) {
-    // TODO(wangtz): handle for kTfLiteInt8
+    case kTfLiteInt8:
+      reference_ops::Negate(GetTensorShape(input), GetTensorData<int8_t>(input),
```

Comment:
it seems this only works when zero_point == 0.

can you handle other cases as well?

thanks

Reply:
Works in what sense?
My assumption was that irregardless of the input being quantized or not, negation is agnostic of the zero-point.
Is this assumption flawed?